src/setMiddleC.js

```diff
@@ -61,13 +66,16 @@ function transposeSingle(note, octave) {
 		index = 2;
 	}
 	let oct = parseInt(note.slice(index,note.length));
+	if(noteIndex==0) {
```

Comment:
There should be a space between `if` and the open parenthesis.. We should bring in proper linting soon. The current linter is just a gulp task that prolly does nothing.